crate/server/src/core/cover_crypt/update_policy.rs

```diff
@@ -28,31 +28,12 @@ pub async fn rekey_keypair_cover_crypt(
     kmip_server: &KMS,
     cover_crypt: Covercrypt,
```

Comment:
Should you really take ownership of this?

Reply:
Changed but it seems that the KMS is designed to create a new Covercrypt object at each call:
```rust
rekey_keypair_cover_crypt(kms, Covercrypt::default(), &owm.id, user, action, params).await
```

Reply:
Yes, you can raise an issue. It may be better to change this.